util/test/demos/main.cpp

```diff
@@ -395,7 +395,7 @@ int main(int argc, char **argv)
   if(argc >= 2 && (!strcmp(argv[1], "--help") || !strcmp(argv[1], "-h") || !strcmp(argv[1], "-?") ||
                    !strcmp(argv[1], "/help") || !strcmp(argv[1], "/h") || !strcmp(argv[1], "/?")))
   {
-    printf(R"(RenderDoc testing demo program
```

Comment:
Please revert these back to printfs - these are not intended to be only debug prints, they are explicitly supposed to be user visible.

Reply:
@baldurk is this OK, or do you still want me to revert them all?

Reply:
Sorry I was away for a long weekend and only just got back.

`DebugPrint` is intended as a debugging log and the stdout output is an implementation coincidence and may go away in future (e.g. if the demos moves to writing to a logfile), so yes I would rather these be reverted to printf.

If you need that test information on android then I think you should create a side-band channel of data separately (e.g. writing the list of tests to a file and pulling that), as logcat is not a reliable medium for transmitting important data because lines may go missing and that could corrupt the output that you're processing.